src/keys/secret.rs

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use crate::JubJubScalar;
```

Comment:
```suggestion
use dusk_jubjub::JubJubScalar;
```
and don't import in 'lib.rs'